README.md

```diff
@@ -261,4 +266,153 @@ Once done, you need to start all components involved, these are:
    Note: Changes to the model in MPS will not automatically synchronize to the dashboard, you will have to manually reload the model.
 
 
+### How to start UC 1b
+
+*Note: All gradle commands assume you are in the top level folder of this repository.*
+
+To start up the system as described in UC 1b, you first have to have built the entire project:
+
+```
+./gradlew
+```
+
+Once done, you need to start all components involved, these are:
```

Comment:
Shouldn't the user also start MPS to actually see some changes to the model? Some description for that would be good.